lib/pbench/test/unit/agent/test_utils.py

```diff
@@ -1,12 +1,14 @@
 """Tests for the utils module.
 """
-
 import os
 import pathlib
+import socket
 import pytest
 import signal
 import time
 
+import ifaddr
```

Comment:
Yes; I *knew* you'd notice that, but *I* didn't notice until after I'd pushed the commit. I have no idea why `vscode` can't get this right, but it doesn't, and sometimes I forget to check on it. (Le sigh.)

Reply:
I figured I would point it out, just in case you ended up going back in to fix something else (which I'm hoping will happen once Peter looks at the missing `else`).